docs/pages/reference/session-recording.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Teleport Session Recording
+description: An overview of Teleport's session recording and its configuration
+---
+
+This page provides an overview of how Teleport records sessions and the various
+configuration options that apply to session recordings. The content that is
+captured depends on the type of the session.
+
+- SSH sessions: Teleport captures the entire pseudo-terminal (PTY) output
+  of the session
+- Kubernetes sessions: Teleport captures the entire PTY output for
+  `kubectl exec` invocations.
+- Desktop sessions: Teleport captures the contents of the desktop screen
+  and any mouse input. Teleport does not capture keystrokes in the remote
+  desktop.
+- App sessions: Teleport captures a stream of `app.session.request`
+  audit events related to application access. The audit events are
+  bucketed into 5-minute time intervals informally referred to as "chunks."
+- Database sessions: Teleport captures a stream of audit events
+  related to the database being accessed.
+
+Teleport supports two different recording configurations: synchronous and
```

Comment:
```suggestion
## Session recording configurations

Teleport supports two different recording configurations: synchronous and
```
Suggested because this is a different topic area than the earlier breakdown of what session recording captures